RELEASE.md

```diff
@@ -0,0 +1,81 @@
+Release Instructions
+====================
+
+Set up Sonatype Account
+-----------------------
+* Sign up for a Sonatype JIRA account [here](https://issues.sonatype.org)
+* Click *Sign Up* in the login box, follow instructions
+
+Get access to repository
+------------------------
+* Go to [community support](https://issues.sonatype.org/browse/OSSRH)
```

Comment:
When you navigate here, it is not clear how to create a new issue. Perhaps we could list more specific steps for how to create the request.